rotkehlchen/data_import/importers/bitcoin_tax.py

```diff
@@ -0,0 +1,337 @@
+            event_subtype=HistoryEventSubType.RECEIVE,
+        )
+        events.extend([spend_event, receive_event])
+        if fee_amount != ZERO and fee_asset is not None:
+            fee_event = HistoryEvent(
+                event_identifier=event_identifier,
+                sequence_index=2,
+                timestamp=timestamp,
+                location=location,
+                asset=fee_asset,
+                balance=Balance(
+                    amount=fee_amount,
+                    usd_value=ZERO,
+                ),
+                notes=memo,
+                event_type=event_type,
+                event_subtype=HistoryEventSubType.FEE,
+            )
+            events.append(fee_event)
+        self.add_history_events(cursor, events)
+
+    def _consume_income_spending_event(
+            self,
+            cursor: DBCursor,
+            csv_row: dict[str, Any],
+            event_identifier: str,
+            timestamp: TimestampMS,
+            location: Location,
+            action: str,
+            asset: AssetWithOracles,
+            asset_amount: AssetAmount,
+            fee_asset: Optional[AssetWithOracles],
+            fee_amount: Fee,
+            memo: str,
+    ) -> None:
+        try:
+            event_type, event_subtype = self.ACTION_TO_HISTORY_EVENT_TYPE[action]
+        except KeyError as e:
+            raise UnsupportedCSVEntry(
+                f'Unsupported entry {action}. Data: {csv_row}') from e
+
+        events: list[HistoryBaseEntry] = []
+        event = HistoryEvent(
+            event_identifier=event_identifier,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=location,
+            asset=asset,
+            balance=Balance(
+                amount=asset_amount,
+                usd_value=ZERO,
+            ),
+            notes=memo,
+            event_type=event_type,
+            event_subtype=event_subtype,
+        )
+        events.append(event)
+        if fee_amount != ZERO and fee_asset is not None:
+            fee_event = HistoryEvent(
+                event_identifier=event_identifier,
+                sequence_index=1,
+                timestamp=timestamp,
+                location=location,
+                asset=fee_asset,
+                balance=Balance(
+                    amount=fee_amount,
+                    usd_value=ZERO,
+                ),
+                notes=memo,
+                event_type=HistoryEventType.SPEND,
+                event_subtype=HistoryEventSubType.FEE,
+            )
+            events.append(fee_event)
+        self.add_history_events(cursor, events)
+
+    def _consume_event(
+            self,
+            cursor: DBCursor,
+            csv_row: dict[str, Any],
+            csv_type: str,
+            timestamp_format: str = '%Y-%m-%d %H:%M:%S %z',
+    ) -> None:
+        """
+        Consume the file containing trades from BitcoinTax.
+        - UnknownAsset
+        - DeserializationError
```

Comment:
I would state in the docstring that this and all other relevant functions in this file can raise `UnsupportedCSVEntry` errors